variables.sh

```diff
@@ -7,7 +7,7 @@ GRPC_JAVA_VERSION=${GRPC_JAVA_VERSION:-1.34}
 GRPC_WEB_VERSION=${GRPC_WEB_VERSION:-1.2.1}
 GRPC_GATEWAY_VERSION=${GRPC_GATEWAY_VERSION:-2.0.1}
 GO_VERSION=${GO_VERSION:-1.14}
-BUILD_VERSION=${BUILD_VERSION:-3}
+BUILD_VERSION=${BUILD_VERSION:-4}
```

Comment:
I think you need to pull latest/rebase, the current build version is 4

Reply:
Done. Bumped to 5